FMCA/src/util/Macros.h

```diff
@@ -29,7 +29,8 @@ namespace FMCA {
 #define FMCA_ZERO_TOLERANCE std::numeric_limits<FMCA_SCALAR>::epsilon()
 #define FMCA_BBOX_THREASHOLD FMCA_ZERO_TOLERANCE
 
-typedef FMCA_INDEX Index;
```

Comment:
Please keeep FMCA_INDEX here. This is safer as this is used everywhere else

Reply:
> What changed in the jupyter notebooks? Please fix the Macros.h

Markdown documentation of how to apply pseudo Inverse